### PR TITLE
[#160727627] Remove temporary concourse team cleanup task

### PR DESCRIPTION
**Pull request by alext**
## What

This was required prior to the upgrade, but is no longer required now
that everything has been upgraded.

How to review
-------------

Run the pipeline, verify it applies cleanly - it should be a no-op deploy.

Who can review
--------------

Not me.

### CODE REVIEW

File: concourse/pipelines/create-bosh-concourse.yml [hunk 1]
```diff
@@ -1341,7 +1341,6 @@ jobs:
         - get: bosh-secrets
         - get: concourse-manifest
           passed: ['generate-concourse-config']
-        - get: concourse-secrets
```
Comment (jpluscplusm) [inline concourse/pipelines/create-bosh-concourse.yml:1344]: Good catch.

**Review by jpluscplusm — APPROVED** [review]
LGTM.

File: concourse/pipelines/create-bosh-concourse.yml (hunk 1 above)
Comment (alext) [reply concourse/pipelines/create-bosh-concourse.yml:1344]: I didn't catch it, our concourse linter did...

**Review by richardTowers — APPROVED** [review]
Looks good to me too. Deployed to my pipeline and it looks like a noop as predicted.

**Pull request merged**